test/components/util/util.go

```diff
@@ -477,7 +477,7 @@ func TestNamespace(name string) string {
 
 // IsUserNamespace checks for user namespace.
 func IsUserNamespace(ns string) bool {
-	if ns == "kube-system" || ns == "kube-public" || ns == "kube-node-lease" || ns == "default" {
+	if ns == "kube-system" || ns == "kube-public" || ns == "kube-node-lease" || ns == "default" || ns == "lokomotive-system" { //nolint:lll
```

Comment:
this will become shorter and more readable with switch case.

Reply:
Cool, let me update it